### PR TITLE
LEAN-2035 (update react-select)

### PR DESCRIPTION
**Pull request by asouqi**

### CODE REVIEW

**Comment by asouqi** [comment]
@mnatsheh do we have a green light to update `react-select` for other repo(`manuscripts-body-editor`, `manuscripts-comment-editor`, `manuscripts-comment-editor`) or that will be done in another ticket?

**Comment by mnatsheh** [comment]
@asouqi yes, please go ahead.

File: src/components/projects/CategorisedKeywordsInput.tsx [hunk 1]
```diff
@@ -316,7 +316,7 @@ export const CategorisedKeywordsInput: React.FC<{
       <OptionWrapper
         focused={keywordToEdit ? data._id === keywordToEdit._id : false}
         ref={innerRef}
-        {...innerProps}
+        {...(innerProps as any)}
```
Comment (mbartenev-atypon) [inline src/components/projects/CategorisedKeywordsInput.tsx:319]: any chance we can do a more specific type here?

**Review by mbartenev-atypon — APPROVED** [review]

File: src/components/projects/CategorisedKeywordsInput.tsx (hunk 1 above)
Comment (asouqi) [reply src/components/projects/CategorisedKeywordsInput.tsx]: updated the ref to be null to ignore the ref of innerProps so with that no need for the `any`. 

and I used it as there is a type mismatch with the ref of styled-components dependency and the one we get from `react-select`. in the new version of [react-select](https://github.com/JedWatson/react-select/blob/master/docs/styled-components.tsx) they remove that dependency and use inline CSS to style custom component!

**Pull request merged**